packages/design-system-web/src/components/geov-map-places/geov-map-places.tsx

```diff
@@ -0,0 +1,293 @@
+import { Component, h, Host, Prop, State, Element } from '@stencil/core';
+import { isNode } from '../../lib/isNode';
+import { importMapLibre } from '../../lib/importMapLibre';
+import { SparqlBinding, SparqlRes, sparqlJson } from '../../lib/sparqlJson';
+import { flag } from 'ionicons/icons';
+import { LngLatBounds, Popup } from 'maplibre-gl';
+
```

Comment:
Import type to avoid bundling of maplibre-gl with webcomponent.
```suggestion
import type { LngLatBounds } from 'maplibre-gl';
```